drsm/drsm.go

```diff
@@ -89,8 +86,6 @@ func (d *Drsm) ConstuctDrsm(opt *Options) {
 	d.podDown = make(chan string, 10)
 	d.scanChunks = make(map[int32]*chunk)
 	d.globalChunkTblMutex = sync.Mutex{}
-	t := time.Now().UnixNano()
-	rand.Seed(t)
```

Comment:
Why you are removing seed? It is useful.

Reply:
From `staticcheck` repo (that is not covered by `golangci-lint`:
```
drsm/drsm.go:93:2: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.  (SA1019)
```